Readme.md

```diff
@@ -22,10 +22,18 @@ The full documentation of Tracee is available at [https://aquasecurity.github.io
 
 Before you proceed, make sure you follow the [minimum requirements for running Tracee](https://aquasecurity.github.io/tracee/dev/install/prerequisites/).
 
+If running on BTF enabled kernel:
+
 ```bash
-docker run --name tracee --rm --privileged -v /lib/modules/:/lib/modules/:ro -v /usr/src:/usr/src:ro -v /tmp/tracee:/tmp/tracee -it aquasec/tracee:latest
+docker run --name tracee --rm --privileged -it aquasec/tracee:latest
 ```
 
+If running on BTF disabled kernel:
+
+```bash
+docker run --name tracee --rm --privileged -v /lib/modules/:/lib/modules/:ro -v /usr/src:/usr/src:ro -v /tmp/tracee:/tmp/tracee -it aquasec/tracee:latest
+
+```
```

Comment:
IMO remove from the quickstart, and link to the doc If non-btf

Reply:
Which doc do you think should be linked to?

Reply:
Perhaps a new document that talks about the differences between CO:RE and kernel-header-dependent versions of tracee